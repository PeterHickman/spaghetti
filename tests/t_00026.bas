10 REM Test the INT funtion
20 let f = 1.0
30 gosub 170
40 let f = 1.4
50 gosub 170
60 let f = 1.5
70 gosub 170
80 let f = 1.7
90 gosub 170
100 let f = 1.9999
110 gosub 170
120 let f = 1.9999999999
130 gosub 170
140 let f = 2.0000001
150 gosub 170
160 end
170 let i = int(f)
180 if i = 1.0 goto 210
190 print "Failed"
200 goto 220
210 print "Ok"
220 return
