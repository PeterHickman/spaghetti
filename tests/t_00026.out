Ok
Ok
Ok
Ok
Ok
Ok
Failed
